app/controllers/items_controller.rb

```diff
@@ -1,4 +1,27 @@
 class ItemsController < ApplicationController
+  before_action :authenticate_user!, except: [:index, :show]
+
   def index
+    @items = Item.all
```

Comment:
```suggestion
    #@items = Item.all
```
こちらは商品一覧機能ブランチでの実装になるので、上記のようにコメントアウトしておきましょう。
  理由　今後のコードレビュー時に差分を反映させるため